knowledge-base/remove-selection-indicator-collectionview-windows.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Removing Selection Indicator from CollectionView on Windows
+description: Learn how to remove the selection indicator from a CollectionView in a .NET MAUI application running on Windows.
+type: how-to
+page_title: How to Remove the Selection Indicator from RadCollectionView in Windows
+slug: remove-selection-indicator-collectionview-windows
+tags: collectionview, .net maui, windows, radcollectionviewitemview, selection, indicator, control template, style
+res_type: kb
+---
+
+## Environment
+
+| Product | CollectionView for .NET MAUI |
+| --- | --- |
+| Version | 7.1.0 |
```

Comment:
I thought we usually included the author for MAUI as well. Is this no longer the case?

Reply:
yes, I forgot to add it. now it is okay. Thank you